lms/djangoapps/verify_student/views.py

```diff
@@ -1175,15 +1179,69 @@ def post(self, request, course_id, checkpoint_name):
                 EVENT_NAME_USER_SUBMITTED_INCOURSE_REVERIFY, user.id, course_id, checkpoint_name
             )
 
-            return HttpResponse()
+            redirect_url = self._get_redirect_url(course_id, location)
+            return JsonResponse({
+                'url': redirect_url
+            })
+        except Http404 as e:
+            log.exception("Invalid location during photo verification.")
+            return HttpResponseBadRequest(_(e.message))
         except IndexError:
             log.exception("Invalid image data during photo verification.")
             return HttpResponseBadRequest(_("Invalid image data during photo verification."))
         except Exception:  # pylint: disable=broad-except
-            log.exception("Could not submit verification attempt for user {}.").format(request.user.id)
+            log.exception("Could not submit verification attempt for user {}.".format(request.user.id))
             msg = _("Could not submit photos")
             return HttpResponseBadRequest(msg)
 
+    def _get_redirect_url(self, course_id, location):
```

Comment:
This code looks like it's copied from one of the views in the `courseware`.  I'd prefer to put this code in one place that can be used both by `courseware` and by `verify_student`, but I'm not sure where that would be.  @jimabramson what would you suggest?